CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Compatibility:
 * Set `RbConfig::CONFIG['archincludedir']` (#3396, @andrykonchin).
 * Support the index/length arguments for the string argument to `String#bytesplice` added in 3.3 (#3656, @rwstauner).
 * Implement `rb_str_strlen()` (#3697, @Th3-M4jor).
+* Implement `rb_enc_interned_str()` (#3706, @Th3-M4jor).
```

Comment:
Shouldn't it be #3703?

Reply:
It should, we use issue numbers in the changelog, not PR numbers.